ansible/playbooks/etcd-setup.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Why is this at the top level playbooks directory, will each role have a setup play here? Are you envisioning having a `deploy-cluster.yml` script that would call `<role>-setup.yml`. To expand we could have a `reset-cluster.yml` call `<role>-reset.yml's` In each `<role>-reset.yml` we use the appropriate tags or includes to enforce the right order and control of tasks executed within that role.

We need to figure out the exactness of how to run something like `etcd-setup.yml` or an `etcd-upgrade.yml`. Maybe we just call specific playbooks in the roles themselves tasks with these alternate flows. We can just call the role with an upgrade tag. I think the later would be easiest to do with what ansible provides and what is existing in contrib.